test/rpc/test_rpc_class.py

```diff
@@ -80,4 +80,5 @@ def test_get_class_at(deploy_info):
 
     assert contract_class["entry_points_by_type"] == EXPECTED_CONTRACT_CLASS
     assert isinstance(contract_class["program"], str)
-    decompress_program({"contract_class": contract_class}, False)
\ No newline at end of file
+    decompress_program({"contract_class": contract_class}, False)
+    
```

Comment:
I still see something reported here

Reply:
This commit introduced the required change: https://github.com/Shard-Labs/starknet-devnet/pull/227/commits/ebc7f2f99a15e8a6ee8d28db5fbe5e5a9335bd23
This commit undid the change: https://github.com/Shard-Labs/starknet-devnet/pull/227/commits/6be3079bb5cd839444feaf89d17751a79a63a998